### PR TITLE
Add Astarte Device SDKs page

### PR DESCRIPTION
**Pull request by bettio**

### CODE REVIEW

File: doc/pages/user/041-astarte_device_sdks.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Astarte Device SDKs allow connecting any device to an Astarte instance.
+
+Astarte Device SDKs are ready to use libraries that provide communication and pairing primitives.
+
+Under the hood Astarte Device SDKs make use of MQTT, BSON, http, persistence and crypto libraries
+to implement [Astarte MQTT v1 Protocol](080-mqtt-v1-protocol.html) and all the other
+useful features.
+
+Virtually an SDK is not required to connect an application to Astarte using MQTT, but it enables
```
Comment (rbino) [inline doc/pages/user/041-astarte_device_sdks.md:13]: I'd remove "Virtually" here

File: doc/pages/user/041-astarte_device_sdks.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+rapid development and a pleasant developer experience.
+
+They can be easily integrated into new or existing IoT projects written in any of the supported
+languages or platforms.
+At the moment the following SDKs are available:
+* C
+  * ESP32
```
Comment (rbino) [inline doc/pages/user/041-astarte_device_sdks.md:20]: Link to ESP32 SDK

File: doc/pages/user/041-astarte_device_sdks.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* Elixir: [astarte-device-sdk-elixir](https://github.com/astarte-platform/astarte-device-sdk-elixir)
+* Go: [astarte-device-sdk-go](https://github.com/astarte-platform/astarte-device-sdk-go)
+* Java
+  * Android: [astarte-device-sdk-java](https://github.com/astarte-platform/astarte-device-sdk-java)
+  * Generic: [astarte-device-sdk-java](https://github.com/astarte-platform/astarte-device-sdk-java)
+* Python: [astarte-device-sdk-python](https://github.com/astarte-platform/astarte-device-sdk-python)
+* C++
```
Comment (rbino) [inline doc/pages/user/041-astarte_device_sdks.md:27]: Are you using alphabetical order? If so, C++ must be moved up

**Review by rbino — CHANGES_REQUESTED** [review]

File: doc/pages/user/041-astarte_device_sdks.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Astarte Device SDKs
+
+## Introduction
+
+Astarte Device SDKs allow connecting any device to an Astarte instance.
+
+Astarte Device SDKs are ready to use libraries that provide communication and pairing primitives.
+
+Under the hood Astarte Device SDKs make use of MQTT, BSON, http, persistence and crypto libraries
```
Comment (rbino) [inline doc/pages/user/041-astarte_device_sdks.md:9]: I think that also HTTP should be uppercase

**Review by rbino — CHANGES_REQUESTED** [review]

**Comment by bettio** [comment]
This PR has been continued from @Annopaolo, let's close it.

**Pull request closed without merging**